QtlTodoList/Screen/TodoListDetails/AddTodoView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AddTodoView.swift
+//  QtlTodoList
+//
+//  Created by 柿崎逸 on 2023/06/17.
+//
+
+import SwiftUI
+
+struct AddTodoView: View {
+    // MARK: - Property Wrappers
+    @State private var title = ""
+    @State private var message = ""
+    @Binding var isTodoAddDetails: Bool
+    @StateObject private var todoListViewModel = TodoListViewModel.shared
+    // MARK: - body
```

Comment:
```suggestion
    @StateObject private var todoListViewModel = TodoListViewModel.shared
   
     // MARK: - body
```

[imo ]MARKコメントの上は空行を挟んだ方が読みやすくなるかなと

Reply:
こちらのファイル移動を行ったので削除しました🙇‍♂️
ファイル移動した時に修正しました！
a18c31f199a0f34de350af23ee0b229fb28e9e81